final_task/rss_reader/models.py

```diff
@@ -0,0 +1,48 @@
+""" Data models module """
+import argparse
+
+
+class ArgReader:
```

Comment:
Why do you need a wrapping class for `parser`? You could just make a function just like __init__() method here and return parser object

Reply:
fixed